### DISCUSSION TITLE
Contact Binary overflow when computing

### DISCUSSION THREAD

**Discussion by lancelot49** [phoebe-project/phoebe2#802] [Questions]
Hello,

I am currently trying to  work with a binary system, that is very likely a contact binary, trying things out with "contact_binary=True". I estimated few parameters with the rv_geometry estimator, but in trying to compute a model after adopting the rv solution, I get that both the primary and the secondary are overflowing at L2/L3. Any idea how to approach this, especially since requiv are being constrained by pot,q, and sma? Any help will be greatly appreciated.

 Context:
"ValueError: failed to pass checks
Run Checks Report: FAIL
ERROR: primary is overflowing at L2/L3 (requiv=1.5, requiv_min=1.1527793662130486, requiv_max=1.3477729064528805) (3 affected parameters, affecting system,run_compute)
ERROR: secondary is overflowing at L2/L3 (requiv=1.4999999999999996, requiv_min=0.7986360329466896, requiv_max=1.0171859977532547) (3 affected parameters, affecting system,run_compute)
ERROR: constraints ['pot@contact_envelope@envelope@constraint'] failed to run.  Address errors and try again.  Call run_failed_constraints to see the tracebacks. (1 affected parameters, affecting system,run_compute)"

**Comment by kecnry** [comment]
The estimators are not aware of the constraints or physical or technical boundaries of the parameter space, but rather just propose values based on whatever algorithm they use (this is by design so that they can run as efficiently as possible to give a reasonable starting place, but are not intended to be fully automated).  In this case, you can either choose not to adopt anything, adopt everything but the radius (for example), or set the radius manually later to something within the accepted range between `requiv_min` and `requiv_max`.
